python-checks/src/test/resources/checks/unreadPrivateMethods.py

```diff
@@ -0,0 +1,28 @@
+class A:
```

Comment:
Maybe we could have a test for an unused method with 1 leading underscore to ensure we don't report any issues on those.